apps/pipelines/migrations/0005_auto_20240802_0039.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.14 on 2024-08-02 00:39
+
+from django.db import migrations
+
+from apps.pipelines.flow import Flow
+
+
+def _migrate_nodes(apps, schema_editor):
+    Pipeline = apps.get_model("pipelines", "Pipeline")
+    Node = apps.get_model("pipelines", "Node")
+    for pipeline in Pipeline.objects.all():
+        flow = Flow(**pipeline.data)
+        for node in flow.nodes:
+            node_object, _ = Node.objects.get_or_create(pipeline=pipeline, flow_id=node.id)
+            node_object.type = node.data.type
+            node_object.params = node.data.params
+            node_object.label = node.data.label
+            node_object.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("pipelines", "0004_node"),
+    ]
+
+    operations = [migrations.RunPython(_migrate_nodes, reverse_code=migrations.RunPython.noop)]
```

Comment:
nit: can't we just delete all `Node` objects in the down migration?

Reply:
https://github.com/dimagi/open-chat-studio/pull/575/commits/707b5c7391b471d9a8b92c48dbf1be649107b3c3